Community-Guide.md

```diff
@@ -4,34 +4,23 @@
 # Egeria Community Guide
 
 This project welcomes contributors from any organization or background, provided they are

```

Comment:
The link https://odpi.github.io/egeria-docs/guides/community/ seems to already have this information